std/manual.md

```diff
@@ -1210,6 +1210,61 @@ console.log(result["/foo.ts"].map);
 We would expect the `enum` would be rewritten to an IIFE which constructs the
 enumerable, and the map to be defined.
 
+## TypeScript Compiler Options
+
+In Deno ecosystem, all strict flags are enabled in order to comply with TypeScript
```

Comment:
I think you will need to format this `./tools/format.py`

Reply:
I thought that it would not affect Markdown, but CI already failed because of this😓

Reply:
Formatted